lib/RAST_SDK/RAST_SDKImpl.pm

```diff
@@ -740,7 +745,7 @@ sub annotate {
 		$message .= " Of the original features, ".$functionchanges." were re-annotated by RAST with new functions.";
 	}
 	$message .= " Overall, a total of ".$seedfunctions." genes are now annotated with ".keys(%{$genomefunchash})." distinct functions. Of these functions, ".keys(%{$seedfunchash})." are a match for the SEED annotation ontology.";
-
+	print($message);
```

Comment:
What was happening to this before?  Was it just being lost?

Reply:
This was still in the report object but I think it's helpful to see when running unit tests too